science/shtools/Portfile

```diff
@@ -36,10 +36,10 @@ variant openmp description {Add OpenMP support} {
 
 test.run            yes
 test.target         fortran-tests-no-timing
-test.args           PREFIX=${prefix} F95="${prefix}/bin/gfortran-mp-10" LAPACK="-framework Accelerate" BLAS=""
+test.args           PREFIX=${prefix} F95="${prefix}/bin/gfortran-mp-10" LAPACK="-lopenblas" BLAS=""
 
 destroot.args       PREFIX=${prefix}
 
 depends_build       port:gcc10
 depends_test        port:gcc10
-depends_lib         port:fftw-3
+depends_lib         port:fftw-3 port:openblas
```

Comment:
Please put this new dependency on openblas on the next line.

Reply:
thanks! done...